package.json

```diff
@@ -37,7 +37,7 @@
   },
   "dependencies": {
     "@fontsource/jetbrains-mono": "^4.5.11",
-    "@nasa-jpl/stellar": "^1.0.10",
+    "@nasa-jpl/stellar": "^1.0.13",
```

Comment:
We can safely remove the dependency update now because of https://github.com/NASA-AMMOS/aerie-ui/pull/178